package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "sort": "./bin/sort words.json > sorted.json && mv sorted.json words.json",
     "lint": "jsonlint -q words.json",
-    "test": "npm run lint && tape test.js"
+    "test": "npm run lint && tape test.js",
+    "validate": "npm ls"
```

Comment:
@rafaelrinaldi looks like NPM or tape added this line. So, I still believe that dependency is not necessary in this case, but the PR #20 at least have a very small dependency...but, why you prefer a dependency rather than a 4 lines code?